resources/css/_code.css

```diff
@@ -15,6 +15,15 @@ pre code.torchlight .summary-caret {
     @apply mr-1;
 }
 
+li code,
+p code {
+    @apply inline-flex max-w-full overflow-scroll
+}
+
```

Comment:
The gap beneath these looks pretty weird, and I'm unsure where it's coming from. I'm also not a fan of how it creates a full-width rectangle:

![image](https://github.com/laravel/bootcamp.laravel.com/assets/4977161/4ed5b115-45be-4fab-a0a2-0173860ee59d)

I think I'd prefer to just change `overflow-wrap` so that it will break mid-word when necessary:

```suggestion
li code,
p code {
    @apply break-words;
}
```

![image](https://github.com/laravel/bootcamp.laravel.com/assets/4977161/cf34a98f-6687-4b7f-89b1-e29b09879f0d)

Reply:
I agree with you, I did the changes as requested in my latest push. Please let me know what do you think.